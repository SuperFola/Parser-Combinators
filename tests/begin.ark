(begin 1 2 3)
(begin)
(begin


(let a 1)
)

(begin (let b 2) 3)

{}
{
1
}

{(let c 4)}
{(if 5 6 7)(mut d 8)}
