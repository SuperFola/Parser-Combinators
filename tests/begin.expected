( Keyword:begin Number:1 Number:2 Number:3 )
( Keyword:begin )
( Keyword:begin ( Keyword:let Symbol:a Number:1 ) )
( Keyword:begin ( Keyword:let Symbol:b Number:2 ) Number:3 )
( Keyword:begin )
( Keyword:begin Number:1 )
( Keyword:begin ( Keyword:let Symbol:c Number:4 ) )
( Keyword:begin ( Keyword:if Number:5 Number:6 Number:7 ) ( Keyword:mut Symbol:d Number:8 ) )